domain/src/test/java/org/oppia/android/domain/locale/LanguageConfigRetrieverProductionTest.kt

```diff
@@ -55,21 +55,29 @@ class LanguageConfigRetrieverProductionTest {
   }
 
   @Test
-  fun testLoadSupportedLanguages_hasThreeSupportedLanguages() {
+  fun testLoadSupportedLanguages_hasSixSupportedLanguages() {
     val supportedLanguages = languageConfigRetriever.loadSupportedLanguages()
 
     // Change detector test to ensure changes to the configuration are reflected in tests since
     // changes to the configuration can have a major impact on the app (and may require additional
     // work to be done to support the changes).
-    assertThat(supportedLanguages.languageDefinitionsCount).isEqualTo(3)
+    assertThat(supportedLanguages.languageDefinitionsCount).isEqualTo(5)
```

Comment:
Good catch! Will fix the name in the latest changes. It was 6 before because I mistakenly enabled Swahili for prod builds. I forgot to update the name when fixing the configuration & test.

So this isn't a case where we're testing a generated proto enum (where there are extra cases). We're actually testing the loaded proto from here: https://github.com/oppia/oppia-android/blob/develop/config/src/java/org/oppia/android/config/productionlanguages/supported_languages.textproto. With this PR, there are now 5 language configurations supported in prod versions of the app hence the change to the test (as a quick parity check, we're adding Arabic and Nigerian Pidgin to the prod languages support, so it makes sense why this is going from 3 to 5).

Reply:
Thanks for explaining.